### PR TITLE
Make sure the service name is the same everywhere

### PR DESCRIPTION
**Pull request by svdgraaf**
The naming convention differs. Sometimes with capital, sometimes without. It's better to have them consistent, as this makes things more predictable.

Merging this PR will make everything use the central service designator, which will always be consistent.

### CODE REVIEW

File: lib/cloudWatch.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default class CloudWatch {
 
   constructor() {
     this.cloudwatch = new AWS.CloudWatch({ region: process.env.REGION || 'eu-central-1' });
-    this.namespace = 'Coinboss';
+    this.namespace = process.env.SERVICE_NAME;
```
Comment (nicka) [inline lib/cloudWatch.js]:
```js
this.namespace = process.env.SERVICE_NAME || 'coinboss'; 
```

**Review by nicka — CHANGES_REQUESTED** [review]
Also be sure to run `yarn test`

File: lib/cloudWatch.js (hunk 1 above)
Comment (svdgraaf) [reply lib/cloudWatch.js]: Done

**Comment by svdgraaf** [comment]
@nicka updated :)

**Review by nicka — APPROVED** [review]

**Comment by tvb** [comment]
@svdgraaf @nicka This change actually limits the way to push different metrics (like from different exchanges) to different namespaces. It is set to to `service: ${file(package.json):name}` which is 
```
{
  "name": "coinboss",
...
```
Also, it is set to all lowercase while all namespaces in CloudWatch are with a capital.

**Pull request merged**